app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -40,6 +40,7 @@ import { RadioGroup } from 'componentLibrary/radioGroup';
 import { EMAIL } from 'common/constants/pluginNames';
 import { FieldTextFlex } from 'componentLibrary/fieldTextFlex';
 import { ruleField } from 'pages/inside/projectSettingsPageContainer/content/notifications/propTypes';
+import { capitalizeWord } from 'pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/util';
```

Comment:
For local component utils, please use relative paths to easier understand its location in the future.
```suggestion
import { capitalizeWord } from './utils';
```